CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+
```

Comment:
Please remove any changes from changelog

Reply:
@varnastadeus will be good?